transfers/views/studentHomeView.py

```diff
@@ -0,0 +1,28 @@
+from django.shortcuts import render
+from django.views import generic
+
+from transfers.forms import PS2TSTransferForm, TS2PSTransferForm
+
+class studentHomeView(generic.TemplateView):
+
+    def redirect(self):
+        return HttpResponseRedirect('/transfers/redirect')
+
+    def get(self, request, *args, **kwargs):        
+        return render (request, "transfers/studentHome.html")
+
+class PS2TSFormView(generic.FormView):
+    form_class = PS2TSTransferForm
+    initial = {'key': 'value'}    
+
+    def get(self, request, *args, **kwargs):
+        form = self.form_class(initial=self.initial)
+        return render(request,  {'form': form})
+
+class TS2PSFormView(generic.FormView):
+	form_class = TS2PSTransferForm
+	initial = {'key': 'value'}
+	
+	def get(self, request, *args, **kwargs):
+		form = self.form_class(initial=self.initial)
+		return render(request, {'form': form})
```

Comment:
There are two files now that I have merged https://github.com/PS-Division-BITS/TMS/pull/16